code/modules/surgery/organs/subtypes/slime.dm

```diff
@@ -0,0 +1,7 @@
+/obj/item/organ/internal/heart/slime
+	name = "osmatic pressure regulator"
+	desc = "A wet squishy heart."
```

Comment:
Should probably have a better description, similar the membrane.

Reply:
Not sure what i'd give it, I went off reference of the other races' descriptions and they didn't have much detail as well(looked at skrell and vox)

Reply:
how about "it appears to be some kind of bilogical pump that uses osmotic pressure to regulate water flow."

also, yeah `osmotic`, not `osmatic`